airflow/configuration.py

```diff
@@ -411,6 +411,7 @@ def run_command(command):
 dags_folder = {TEST_DAGS_FOLDER}
 plugins_folder = {TEST_PLUGINS_FOLDER}
 base_log_folder = {AIRFLOW_HOME}/logs
+entrypoint_group = airflow_test
```

Comment:
This should probably not be hardcoded, perhaps something like `{TEST_ENTRYPOINT_GROUP}`, and should be filled in later in this file just like `{TEST_DAGS_FOLDER}`, etc.